README.md

```
@@ -12,24 +12,27 @@ It's currently a thin wrapper around the OpenAI Realtime speech API, however the
 
 If this sounds interesting, [⭐️ the project on GitHub](https://github.com/vibemachine-labs/arty/stargazers) to help it grow.
 
-## 🎥 Demo Reel (80 seconds)
+## 🎤 Demo audio - browse hacker news (1 min 30 secs)
 
-(Does not play directly, this links to a Vimeo.  Open in a new tab.)
-
-<a href="https://vimeo.com/manage/videos/1127547235" target="_blank">
-  <img src="https://github.com/user-attachments/assets/e500b199-f0a2-4d6b-9a1a-f572fc48361a" alt="Demo video">
-</a>
+https://github.com/user-attachments/assets/0c735cc8-317c-407a-8db1-0f02b65533ae
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)